File: ash/ash2/Dao/Interf/ITb_userDao.cs
using System;
using System.Linq;
using System.Text;
using model;
using System.Collections;

namespace ash2.Dao.Interf
{
    public interface ITb_userDao
    {
        Tb_user get(int id);

        void save(Tb_user model);

        void update(Tb_user model);

        void delete(Tb_user model);

        IList getAll();
    }
}

File: ssh/src/dao/interf/ITb_userDao.java
package dao.interf;

import java.util.List;

import model.test.Tb_user;

public interface ITb_userDao {
	public Tb_user getById(int id);

	public void save(Tb_user model);

	public void update(Tb_user model);

	public void delete(Tb_user model);

	public void deleteById(int id);

	public List getAll();
}